src/main/kotlin/racingcar/Driver.kt

```diff
@@ -1,12 +1,23 @@
 package racingcar
 
-class Driver(private val car: Car) {
-    val driveRecord = mutableListOf<Int>()
+class Driver(val name: String, private val car: Car) {
+
+    val driveRecord = mutableListOf<Int>(car.movement)
```

Comment:
여기저기 불필요한 타입명시가 있는것 같습니다~!
```suggestion
    val driveRecord = mutableListOf(car.movement)
```